docs_user/modules/proc_adopting-the-networking-service.adoc

```diff
@@ -15,19 +18,20 @@ This guide also assumes that:
 
 .Prerequisites
 
-* Previous Adoption steps completed. Notably, MariaDB,{identity_service_first_ref}, and xref:migrating-ovn-data_migrating-databases[Migrating OVN data]
-should be already adopted.
+* Adopt the {identity_service}. For more information, see xref:adopting-the-identity-service_adopt-control-plane[Adopting the {identity_service}]. 
+* Migrate your OVN databases to `ovsdb-server` instances that run in the {OpenShift} cluster. For more information, see xref:migrating-ovn-data_migrating-databases[Migrating OVN data].
+
 
 .Procedure
 //The following link takes me to a 404. Do we need this text? I think we should start the procedure at "Patch OpenStackControlPlane..."
 ifeval::["{build}" != "downstream"]
 As already done for https://github.com/openstack-k8s-operators/data-plane-adoption/blob/main/keystone_adoption.md[Keystone], the Neutron Adoption follows the same pattern.
 endif::[]
 
-* Patch `OpenStackControlPlane` to deploy {networking_service}:
+* Patch the `OpenStackControlPlane` CR to deploy the {networking_service}:
```

Comment:
@slawqo Which files is being updated in this step? Is it the openstack_control_plane.yaml file, which defines the OpenStackControlPlane CR?

Reply:
basically what is updated in this step is the OpenStackControlPlane CR which is already in the OpenShift. You can do it in many different ways, like e.g.:
1. updating openstack_control_plane.yaml file and then use `oc apply -f` command to apply that file to the OCP,
2. using `oc edit openstackcontrolplane` command and then update it "live" in the text editor,
3. using `oc patch` command to patch existing CR and change this setting - this method is showed below